### ISSUE TITLE
ms-vscode.js-debug no longer included

### ISSUE THREAD

**Issue by tsmaeder** [eclipse-theia/vscode-builtin-extensions#123]
The ms-vscode.js-debug extension that provides typescript/javascript debugging in VS Code is no longer included in the theia built-in extension pack in the latest version. It used to be included in the built-in extension pack.

**Comment by tsmaeder** [comment]
This extension is central to dog-fooding and included in VS Code out of the box.

**Comment by vince-fugnitto** [comment]
According to the logs when we published it seems that it was [skipped](https://github.com/eclipse-theia/vscode-builtin-extensions/actions/runs/5787611586/job/15684903814#step:9:98).
@marcdumais-work any ideas?

**Comment by marcdumais-work** [comment]
> According to the logs when we published it seems that it was [skipped](https://github.com/eclipse-theia/vscode-builtin-extensions/actions/runs/5787611586/job/15684903814#step:9:98). @marcdumais-work any ideas?

Hummm, It seems this extension and its companion were skipped, when generating the built-ins pack, because they were not found on openvsx. I do not know if it's a bug in our script or if there was a glitch with the server. 


```
Skipping extension, as it's not found in the registry : ms-vscode.js-debug-companion
Skipping extension, as it's not found in the registry : ms-vscode.js-debug
```

**Comment by marcdumais-work** [comment]
I think I found a bug that explains the above. We can fix it now, so that these extensions will be part of the pack next time.

**Comment by marcdumais-work** [comment]
@tsmaeder we found and [fixed the bug](https://github.com/eclipse-theia/vscode-builtin-extensions/pull/124), that mistakenly excluded `ms-vscode.js-debug` and its companion extension, from the vscode built-ins extension pack v1.79.0.  When we next publish a new set of built-ins and a new pack along with it, the extensions should be present again. 

The good news is that Blueprint is unaffected (probably like most Theia apps out- there) It's not yet using that version of the built-ins pack, currently using slightly older v1.77.0. The extensions, listed in a pack, are not versioned, and always resolve to the latest compatible version, according to stated extensions API version supported (the version of the pack does not play a role). This means that, using a relatively recent version of the pack, should be virtually just as good. The only downside I see is that any extension added or legitimately removed in 1.79.0 would not be if we used the previous version. I do not think that's a problem ATM, but if it is for someone, they can work-around the issue by strategically explicitly adding and/or excluding extensions, to fit their needs. 

Plan going forward: For Blueprint, wait until a new set of vscode built-ins is released before updating the pack (currently v1.77.0)

WDYT?
